### PR TITLE
Add a warning msg in the upgrade process

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR adds a warning message in the upgrade process that the underlying snapshots will be deleted.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NOTE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +125,7 @@ The change from v1alpha1 to v1beta1 snapshot APIs is not backward compatible.
 
 If you have already deployed v1alpha1 snapshot APIs and external-snapshotter sidecar controller and want to upgrade to v1beta1, you need to do the following:
 * Delete volume snapshots created using v1alpha1 snapshot CRDs and external-snapshotter sidecar controller.
+  * <strong>Note: The underlying snapshots on the storage system will be deleted after this step!!!</strong>
```
Comment (msau42) [inline README.md]: I think it's also important to point out that if they skip this step, the next step will still delete all the objects and potentially the snapshots too

Comment (xing-yang) [reply README.md]: I'll just move the warning to the beginning to say that snapshots will be deleted to be safe.

**Pull request merged**